std/array.d

```diff
@@ -3891,3 +3891,231 @@ unittest
     assert(appS.data == "hellow");
     assert(appA.data == "hellow");
 }
+
+version(all)
+{
+
+import std.traits : CommonType, Unqual, isStaticArray;
+import std.conv : emplaceRef;
+
+// @nogc: https://issues.dlang.org/show_bug.cgi?id=18439
+//nothrow @safe pure:
+nothrow:
+@safe:
+pure:
+
+/++
+Returns a static array constructed from `a`. The type of elements can be
```

Comment:
needs `Params:` and `Returns:` sections. same throughout

Reply:
done i think